app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -158,5 +158,30 @@ public function processInvitationcall($observer)
             }
         }
     }
+    public function sendSyncUrl(): void {
+        $helper = Mage::helper('webwinkelconnect');
+        if (!$helper->isProductReviewInviteEnabled()) {
+            return;
+        }
+
+        $curl = new Varien_Http_Adapter_Curl();
+        $curl->setConfig([
+            'timeout'   => 10,
+        ]);
+        $url = 'https://dashboard.webwinkelkeur.nl/webshops/sync_url';
+        $data = json_encode([
+            'webshop_id' => $helper->getShopId(),
+            'api_key' => $helper->getApiKey(),
+            'url' => Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_WEB) . 'webwinkelkeur/index/sync',
+        ]);
+
+        $curl->write(Zend_Http_Client::POST, $url, '2', ['Content-Type:application/json'], $data);
+        $response = $curl->read();
+        if (!$response) {
+            $curl->close();
```

Comment:
Resource cleanup (cURL close in this case) that runs even if an exception is used or return statement is used can be achieved more cleanly (without repetition of the cleanup code).

Make sure you read this: https://www.php.net/manual/en/language.exceptions.php

In this case:

~~~php
try {
  // do stuff with $curl
} finally {
  $curl->close();
}
~~~

The finally clause always executes, even if an exception is thrown.